get-started/installation-procedure.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: Installation
 slug: installation
-tags: installation
+tags: installation, accounts, updates
 publish: true
 position: 20
 ---
 
 ## Installation 
  
-Steps to install Fiddler Everywhere client on your system: 
+Steps to install Fiddler Everywhere client on your system (Mac OS, Windows, or Linux): 
```

Comment:
install Fiddler Everywhere client -> "the Fiddler Everywhere client" or just "Fiddler Everywhere"